UnicodeJsps/src/test/java/org/unicode/jsptest/TestBuilder.java

```diff
@@ -20,17 +20,20 @@
 
 import com.ibm.icu.dev.test.TestFmwk;
 
+/**
+ * "Not really a test, move"
+ */
```

Comment:
These comments look temporary. Remove before merging?

Reply:
The comments were on TestAll about these classes. They belong on the classes I think.